tests/func-tests/utils.go

```diff
@@ -48,16 +48,29 @@ func FlagParse() {
 
 func BeforeEach() {
 	virtClient, err := kubecli.GetKubevirtClient()
-	kvtutil.PanicOnError(err)
+	Expect(err).ShouldNot(HaveOccurred())
+
+	Eventually(func() error {
+		return virtClient.RestClient().Delete().Namespace(kvtutil.NamespaceTestDefault).Resource("virtualmachines").Do(context.TODO()).Error()
+	}).WithTimeout(time.Minute).WithPolling(time.Second).Should(Succeed())
```

Comment:
again I would maybe like to see this extracted to a method and have here something like this:

```
deleteAll("virtualmachines")
deleteAll("virtualmachineinstances")
deleteAll("persistentvolumeclaims")
```